ios/RNPhotosFramework/PHCollectionService.m

```diff
@@ -98,7 +98,8 @@ +(NSMutableDictionary *)generateAlbumsResponseFromParams:(NSDictionary *)params
     int numberOfPreviewAssets = [RCTConvert int:params[@"previewAssets"]];
     BOOL includeMetadata = [RCTConvert BOOL:params[@"includeMetadata"]];
     BOOL includeResourcesMetadata = [RCTConvert BOOL:params[@"includeResourcesMetadata"]];
-    
+    BOOL toggleAlbumTypeFix = [RCTConvert BOOL:params[@"toggleAlbumTypeFix"]];
```

Comment:
```suggestion
    BOOL toggleAlbumTypeFix = [RCTConvert BOOL:params[@"albumTypeFix"]];
```

`toggle` adds confusion to what it does. Since this is specifying behaviour, it's best to name your param in such a way that it is clear that this is specifying behaviour